<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PToools</class>
 <widget class="QWidget" name="PToools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PToools</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>压缩后名称：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="openDirBtn">
     <property name="text">
      <string>打开...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="compressedDir_inputLdt"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>压缩  文件：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="compressedFileNamLdt"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="startCompressedBtn">
     <property name="text">
      <string>压缩</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="statusTipLbl">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
